client/components/tooltip/index.tsx

```diff
@@ -62,4 +69,42 @@ const Tooltip: React.FC< TooltipProps > = ( {
 	);
 };
 
-export default Tooltip;
```

Comment:
Tooltip is no longer exposed as the default export.

HoverTooltip and ClickTooltip are now exported from this file.

The intention is to make it more explicit for future devs using these components.